core/common/buffer.hpp

```diff
@@ -222,6 +222,18 @@ namespace kagome::common {
      */
      const std::string_view toString() const;
 
+    /**
+     * @brief return content of bytearray as string
+     * @note Does not ensure correct encoding
+     * @return string
+     */
+    std::string asString() const;
+
+    /**
+     * @brief stores content of a string to byte array
+     */
+    static outcome::result<Buffer> fromString(const std::string &src);
```

Comment:
why result? it never fails

Reply:
For consistency with other ```fromString``` implementations to use uniformly in ```OUTCOME_TRY``` or in template functions.